test/spec/features/modeling/behavior/ResizeBehaviorSpec.js

```diff
@@ -326,6 +326,63 @@ describe('features/modeling - resize behavior', function() {
         })
       );
 
```

Comment:
Style nit-pick: Please separate specs (`it(...)`) from each other with two empty lines.

Reply:
Updated. [q] Can we use es-lint custom rules to enforce this?

Reply:
> [q] Can we use es-lint custom rules to enforce this?

Yes. You could write one, and contribute to [eslint-plugin-bpmn-io](https://github.com/bpmn-io/eslint-plugin-bpmn-io).